SQLServer/MockRepositories/MockIssueRepository.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Abstractions.Models;
+using Abstractions.Repositories;
+using SQLServer.Models;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
Was tempted to wrap all calls in `Task.Run()` but figured it was simpler to let them run synchronously and to disable the warning